declarations/Ask Media Group, LLC.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Ask Media Group, LLC",
```

Indeed, Ask Media Group is the provider, but the service is Ask.com:

> Ask Media Group, LLC ( “AMG”), directly or indirectly operates Ask.com, as well as Reference.com, Life123.com, Consumersearch.com, Shop411.com, and many other sites (the “Sites”) offering search services and everyday useful content (the “Services”).

```suggestion
  "name": "Ask.com",
```